library/src/main/java/org/angmarch/views/NiceSpinner.java

```diff
@@ -123,12 +127,7 @@ public void onRestoreInstanceState(Parcelable savedState) {
             if (bundle.getBoolean(IS_POPUP_SHOWING)) {
                 if (popupWindow != null) {
                     // Post the show request into the looper to avoid bad token exception
-                    post(new Runnable() {
-                        @Override
-                        public void run() {
-                            showDropDown();
-                        }
-                    });
+                    post(this::showDropDown);
```

Comment:
Be sure to add this in the library's `build.gradle` file to enable Java 8 features like passing the method as a reference:
```
compileOptions {
        sourceCompatibility JavaVersion.VERSION_1_8
        targetCompatibility JavaVersion.VERSION_1_8
    }
}
```

Reply:
ok